roles/confluent.test/molecule/zookeeper-mtls/molecule.yml

```diff
@@ -86,6 +86,54 @@ platforms:
     privileged: true
     networks:
       - name: confluent
+  - name: kafka-connect1
+    hostname: kafka-connect1.confluent
+    groups:
+      - kafka_connect
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
```

Comment:
create new fips dockerfile: https://github.com/arhea/docker-fips-library/blob/master/centos/Dockerfile

Reply:
This did not work 
```
% docker run --rm -it fips:latest cat /proc/sys/crypto/fips_enabled
cat: /proc/sys/crypto/fips_enabled: No such file or directory
```

Reply:
testing in docker containers did not work, maybe add an automated test on aws